storage/tianmu/system/cacheable_item.cpp

```diff
@@ -34,7 +34,7 @@ CacheableItem::CacheableItem(char const *owner_name, char const *object_id, int
     // read the configuration parameter
     std::string temp_filename = tianmu_sysvar_cachefolder;
     filename_n_position = temp_filename.length();
-    filename = new char[filename_n_position + 37];  // "...path.../XXXXXXnnnnnnAAAAAAAABBBBBBBB.tianmu_tmp"
+    filename = new char[filename_n_position + 37 + 4];  // "...path.../XXXXXXnnnnnnAAAAAAAABBBBBBBB.tianmu_tmp"
```

Comment:
Pls, would you like use cosntexpr or meaningful words to describe these magic numbers?